content/6_Advanced/Multiplicative.mdx

```diff
@@ -159,16 +181,17 @@ int main() {
 	i--;
 	if (i == n / i) sum -= i * (n / i);
 	printf("%lld\n", sum);
-	return 0;
 }
 ```
 
 </CPPSection>
 </LanguageSection>
-### Focus problem # 3
+
+## Focus Problem 3
```

Comment:
i see there's judge.yousop or whatever links at the bottom- can you do those rq?